server/rooms.ts

```diff
@@ -1238,7 +1238,11 @@ export class BasicChatRoom extends BasicRoom {
 		}
 		if (this.modjoin) {
 			const modjoin = this.modjoin === true ? this.modchat : this.modjoin;
-			message += ` [${modjoin} or higher to join]`;
+			if (this.staffRoom) {
+				message += ` [room staff or higher to join]`;
```

Comment:
????

This seems clearly wrong to me?

Reply:
Really, any room accepts either the modjoin rank in what it's inheriting from, or any local rank.